### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by asoMan1369** [vpnhood/VpnHood#291] [General]
Do you have any documentation to running the application on asp.net for developing ?
I saw your DLL on Nugget. But I need a document for running the application in Visual Studio.

**Comment by trudyhood** [comment]
Yes, you can, but you can not put the request pipeline behind kestrel. VpnHoodServer should listen to the ports explicitly. Here is the sample. No more document exists. Source code support is also so limited.

https://github.com/vpnhood/VpnHood/tree/main/Samples
